### ISSUE TITLE
How to set MD004 to Sublist?

### ISSUE THREAD

**Issue by F1LT3R** [DavidAnson/markdownlint#313]
Hi, great tool. 

I'm having trouble settings sublists.

Here is what I am trying:

```json
{
  "MD004": { "style": "sublist" }
}
```

I would expect this to allow for the following:

```markdown
*  Level 1
  + Level 2
    - Level 3
```

But this gets changed to the following on save:

```markdown
-  Level 1
  - Level 2
    - Level 3
```

What am I doing wrong?

**Comment by DavidAnson** [comment]
That configuration should not report any violations via markdownlint. You can confirm that here: ￼https://dlaa.me/markdownlint/#%25m%23%20Issue%20313%0A%0A*%20Level%201%0A%20%20%2B%20Level%202%0A%20%20%20%20-%20Level%203%0A%0A%3C!--%20markdownlint-configure-file%20%7B%20%22MD004%22%3A%20%7B%20%22style%22%3A%20%22sublist%22%20%7D%20%7D%20--%3E%0A

What editor are you using and what tool, extension, or plug-in is applying those edits when you save the document?￼

**Comment by F1LT3R** [comment]
I see. I think Prettier is overriding things. Thanks.

**Comment by dotku** [comment]
> I see. I think Prettier is overriding things. Thanks.

How did you fix it, with prettier rc or eslintrc?
